api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TransferredInCohortQueries.java

```diff
@@ -201,7 +201,6 @@ public CohortDefinition getPatientsTrfOut() {
     sqlCohortDefinition.setName(
         "Patient Transferred Out With No Drug Pick After The Transferred out Date ");
     sqlCohortDefinition.addParameter(new Parameter("startDate", "Start Date", Date.class));
-    sqlCohortDefinition.addParameter(new Parameter("endDate", "End Date", Date.class));
     sqlCohortDefinition.addParameter(new Parameter("location", "Location", Location.class));
 
     Map<String, Integer> map = new HashMap<>();
```

Comment:
This transferred out cohort is not aligned with the one used in Tx curr. Check the program criteria.
Check which implementation is a aligned with the spec

"Dev: confirm the code is using the same definition of transferred out from Tx Curr ( here it should be by end of previous reporting period.)"